### PR TITLE
test(e2e): update platform e2e specs to run in prod mode

### PR DESCRIPTION
**Pull request by Seamoo13**
#### Please provide a link to the associated issue.
Closes none

#### Please provide a brief summary of this pull request.
This PR updates several platform e2e tests to run in prod mode

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by maxym-valor — APPROVED** [review]

File: e2e/wdio/platform/tests/step-input.e2e-spec.ts [hunk 1]
```diff
@@ -124,18 +124,20 @@ describe('Step input test suite', function() {
     });
 
     // Need to debug on different browsers
-    xit('Verify the value in the field becomes 0 or the minimum if the minimum is larger than 0.', () => {
+    fit('Verify the value in the field becomes 0 or the minimum if the minimum is larger than 0.', () => {
```
Comment (kavya-b) [inline e2e/wdio/platform/tests/step-input.e2e-spec.ts:127]: do we need this to be `fit`?

File: e2e/wdio/platform/tests/time-picker.e2e-spec.ts [hunk 2]
```diff
@@ -93,35 +99,35 @@ describe('Time picker suite', function() {
...
         click(setValidTimeButton);
         expect(doesItExist(errorBorder)).toBe(false);
-        expect(getValue(activeTimePickerInput, 4)).toBe(defaultValidTime);
+        expect(getValue(activeTimePickerInput, 5)).toBe(defaultValidTime);
     });
 
     // skipped due to https://github.com/SAP/fundamental-ngx/issues/4853
     xit('Verify null validity for time picker with reactive form', () => {
-        scrollIntoView(activeTimePickerButton, 9);
+        scrollIntoView(activeTimePickerButton, 10);
```
Comment (kavya-b) [inline e2e/wdio/platform/tests/time-picker.e2e-spec.ts:114]: i see the test has been modified but it is still excluded as `xit`. Is this intentional?

Comment (Seamoo13) [reply e2e/wdio/platform/tests/time-picker.e2e-spec.ts:113]: This was intentional. I updated the selector for activetimePickerButton and knew the index here needed to be updated. That being said, I saw the issue that was blocking this test case has been fixed, so I went ahead and updated this case now.

File: e2e/wdio/platform/tests/step-input.e2e-spec.ts (hunk 1 above)
Comment (Seamoo13) [reply e2e/wdio/platform/tests/step-input.e2e-spec.ts]: fixed

File: e2e/wdio/platform/tests/combobox.e2e-spec.ts [hunk 3]
```diff
@@ -13,7 +15,8 @@ import {
     waitForClickable,
     waitForElDisplayed,
     waitForPresent,
-    waitForUnclickable
+    waitForUnclickable,
+    doubleClick
```
[the remark this answers is not in the document]
Comment (Seamoo13) [reply e2e/wdio/platform/tests/combobox.e2e-spec.ts]: removed

File: e2e/wdio/platform/tests/checkbox.e2e-spec.ts [hunk 4]
```diff
@@ -53,59 +53,59 @@ describe('Checkbox test suite', () => {
 
     describe('check binary checkbox used with form examples', () => {
         // TODO: Unskip after fix
```
[the remark this answers is not in the document]
Comment (Seamoo13) [reply e2e/wdio/platform/tests/checkbox.e2e-spec.ts]: removed

File: e2e/wdio/platform/pages/combobox.po.ts [hunk 5]
```diff
@@ -5,7 +5,9 @@ import {
     scrollIntoView,
     sendKeys,
     waitForPresent,
-    waitForElDisplayed
+    waitForElDisplayed,
+    getText,
```
[the remark this answers is not in the document]
Comment (Seamoo13) [reply e2e/wdio/platform/pages/combobox.po.ts]: removed

File: e2e/wdio/platform/tests/time-picker.e2e-spec.ts (hunk 2 above)
Comment (kavya-b) [reply e2e/wdio/platform/tests/time-picker.e2e-spec.ts:113]: >That being said, I saw the issue that was blocking this test case has been fixed, so I went ahead and updated this case now.
I am a bit confused, so is this test supposed to be `it` now that you updated the testcase or should it still be `xit`?

Comment (kavya-b) [reply e2e/wdio/platform/tests/time-picker.e2e-spec.ts:113]: oh something was wrong with my github i guess, it was showing your old code, please ignore my comment

**Review by kavya-b — APPROVED** [review]

**Review by nikvalor — APPROVED** [review]

**Pull request merged**